Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -92,14 +142,42 @@ pathManagerState& cruisingState::getInstance()
 
 void coordinateTurnElevation::execute(pathManager* pathMgr)
 {
-    //get elevation and turning data
+    
+    SFOutput_t * pSensFusionOutput = sensorFusion::GetSFOutput(); // Get sensor fusion data
+    _WaypointManager_Data_Out * waypointOutput = cruisingState::GetOutputData(); // Get output data from waypoint manager
+    
+    // Set up the coordinated turn function input
+    double track = atan(pSensFusionOutput->latitudeSpeed / pSensFusionOutput->longitudeSpeed) * 180.0 / PI; // Gets angle of track in quadrant 1
+
+    /* 
+    
+    Need a way to convert the value above to a good track value 
+    
+    */
+
+    CoordinatedTurnInput_t turnInput {};
+    turnInput.currentHeadingTrack = track;
+    turnInput.desiredHeadingTrack = waypointOutput->desiredTrack;
+    turnInput.accY = pSensFusionOutput->pitch; // Is this actually the data that you want?
+    
+    // Set up the compute altitude and airspeed function input
+    AltitudeAirspeedInput_t altAirspeedInput {};
+    altAirspeedInput.currentAltitude = pSensFusionOutput->altitude;
+    altAirspeedInput.desiredAltitude = waypointOutput->desiredAltitude;
+    altAirspeedInput.currentAirspeed = pSensFusionOutput->airspeed;
+    altAirspeedInput.desiredAirspeed = 30;                                      // Where am I supposed to get this?
```

Comment:
probably just set a constant that you want to maintain during the cruising phase. Right now, I don't think cruising needs to output variable speed, but the other states will need to. This kinda relates to how we're gonna grab info from the other stages.

Reply:
Will have to decide on a speed :P